cmd/promu.go

```diff
@@ -165,3 +165,12 @@ func stringInSlice(needle string, haystack []string) bool {
 	}
 	return false
 }
+
+func hasRequiredConfigurations(configVars []string) (bool, error) {
```

Comment:
The return values seem to be dependent on each other. What about only returning `error` here. Either it's nil and there is no error or it returns the error message.

```go
if err := hasRequiredConfigurations([]string{"repository.path"}); err != nil {
    fatal(err)
}
```